src/Features/CustomerEffortScoreTracks.php

```diff
@@ -242,4 +266,26 @@ function( $item ) use ( $clear_ces_tracks_queue_for_page ) {
 		);
 		update_option( self::CLEAR_CES_TRACKS_QUEUE_FOR_PAGE_OPTION_NAME, false );
 	}
+
+	/**
+	 * Enqueue the CES survey trigger for setting changes.
+	 */
+	public function run_on_update_options() {
+		if ( $this->has_been_shown( self::SETTINGS_CHANGE_ACTION_NAME ) ) {
+			return;
+		}
+
+		$this->enqueue_to_ces_tracks(
+			array(
+				'action'    => self::SETTINGS_CHANGE_ACTION_NAME,
+				'label'     => __(
+					'How easy was it to update your settings?',
+					'woocommerce-admin'
+				),
+				'pagenow'   => 'woocommerce_page_wc-settings',
+				'adminpage' => 'woocommerce_page_wc-settings',
```

Comment:
Hmm... do we have anywhere where we document how `pagenow` and `adminpage` should be used? I sorta just copy-pasted my settings in #5540, but seeing how you have set these, it makes me question

(a) what they should be set to, and 
(b) if we really need both `pagenow` and `adminpage`

/cc @becdetat

Reply:
Looking at the https://github.com/woocommerce/woocommerce-admin/blob/update/5348-trigger-ces-on-changing-setting/client/customer-effort-score-tracks/customer-effort-score-tracks-container.js#L34, those two variables are used to filter the queue. The values can be retrieved from `window.pagenow` and `window.adminpage`.

Reply:
Thanks!